### DISCUSSION TITLE
Release 1.25

### DISCUSSION THREAD

**Discussion by webspiderteam** [webspiderteam/Bluetooth-DMM-For-Windows#44] [Announcements]
- Fixed When AutoConnect enabled and App minimized app was not restore  showing state #43 
- Renewed app certificate

<hr /><em>This discussion was created from the release <a href='https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/releases/tag/WPFRelease1.25'>Release 1.25</a>.</em>

**Comment by SchnellerReifen** [comment]
Nice program!
is there in future a option to export in svg or something like that? :)

**Reply by webspiderteam** [reply]
Thank you ☺️ I don't think it's necessary to export as SVG or any type of graph because it's already showing graph and also with CSV you can use with Excel to make a graph. Ofcourse that's my opinion. If you have good reason please explain me
